config/base.js

```diff
@@ -86,6 +86,22 @@ const SCHEMA_CONFIG = Joi.object()
     })
     .unknown(false);
 
+const SCHEMA_CONFIG = Joi.object()
```

Comment:
Consider renaming. Keeping it generic would be ambiguous.
```suggestion
const SCHEMA_CONFIG_POST_TEMPLATE_MERGE = Joi.object()
```